### ISSUE TITLE
Issue with autocomplete

### ISSUE THREAD

**Issue by Manuelandro** [jamesfzhang/auto-save#7]
Hi,
i'm having issues with css autocomplete behaviors.( don't know yet if it happens with other languages).
When i type few characters and sublime suggests me the list of tags available, auto-save saves the file and breaks down the suggestion list.

**Comment by csarcom** [comment]
I tried to change it too, and for me nothing happens, still using 1s, I used the same settings file as @Kiura did.

**Comment by jamesfzhang** [comment]
@csarcom Can you open the console (View --> Show Console) and see if there are any errors?

**Comment by csarcom** [comment]
I`m getting this error when editing Preferences.sublime-settings:

```python
reloading Packages/User/Preferences.sublime-settings
Errors parsing theme:
Rule is missing a class name
```

My settings is:

```python
{
	"color_scheme": "Packages/User/primer.light (Flake8Lint).tmTheme",
	"font_face": "Courier New",
	"font_size": 12,
	"ignored_packages":
	[
		"Vintage"
	],
	"theme": "Primer.sublime-theme",
	"auto_save_delay_in_seconds": 5,
}
```

I commented // "theme": "Primer.sublime-theme", and the erros stop, but auto_save_delay still does not change...

**Comment by jamesfzhang** [comment]
@csarcom You have a trailing comma in the last line, which would invalidate this JSON. Try removing that.

**Comment by csarcom** [comment]
Still nothing happens :( Do you think it could be some problem with other plugins/themes?

**Comment by jamesfzhang** [comment]
@csarcom Possibly. I'm guessing it's the Primer theme that's causing problems since you said it works fine if it's commented out?

**Comment by csarcom** [comment]
@jamesfzhang If I commented out primer theme the error on console disappear, but auto_save_delay still does not work as expected.

**Comment by csarcom** [comment]
I will try it on a fresh sublime install without nothing else when I go home.

**Comment by gleb-s** [comment]
Hello!
Every time facing this issue like in first comment too. =(
Its nice plugin, but fixing this issue will make it really outstanding! Thx anyway

**Comment by sinaru** [comment]
I believe this is the same issue [https://github.com/jamesfzhang/auto-save/issues/21]

**Comment by jamesfzhang** [comment]
Should be fixed in 1.0.8
